front-end/src/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.css';
+import DataEntry from './Components/DataEntry';
+// import Home from './Components/Home';
+import Login from './Components/Login';
+import Registration from './Components/Registration';
+import { Route, createBrowserRouter, createRoutesFromElements, RouterProvider, Outlet } from "react-router-dom";
```

Comment:
import like this
import { BrowserRouter as Router, Routes,Route} from 'react-router-dom";

this way we wont need any other things like createBrowserRouter and others